snap/snapcraft.yaml

```diff
@@ -11,6 +20,7 @@ architectures:
 grade: stable
 confinement: classic
 compression: lzo
+license: Proprietary
```

Comment:
Small nitpick: I prefer to have this line right under the other metadata like `website`, instead of at this place.

The block above contains metadata visible to the user, whereas `grade`, `confinement` and `compression` are implementation details of the snap.

You don't have to fix this line, but I just wanted to give my opinion :)